Sources/SwiftSyntaxMacrosTestSupport/Assertions.swift

```diff
@@ -272,10 +272,13 @@ public func assertMacroExpansion(
   let context = BasicMacroExpansionContext(
     sourceFiles: [origSourceFile: .init(moduleName: testModuleName, fullFilePath: testFileName)]
   )
-  let expandedSourceFile = origSourceFile.expand(macros: macros, in: context).formatted(using: BasicFormat(indentationWidth: indentationWidth))
+  let expandedSourceFile = origSourceFile.expand(macros: macros, in: context)
+  let formattedSourceFile = expandedSourceFile.formatted(using: BasicFormat(indentationWidth: indentationWidth))
+
+  XCTAssertFalse(expandedSourceFile.hasError, "Expanded source should not contain any syntax errors")
```

Comment:
What do you think about running `ParseDiagnsoticGenerator` to actually show the syntax errors?

Reply:
I added this but IMO they still don't make much sense without the syntax tree. So I added that in as well.